tests/integration/targets/nxos_static_routes/tests/common/parsed.yml

```diff
@@ -13,14 +13,11 @@
     state: parsed
   register: result
 
-- ansible.builtin.assert:
+- name: Assert parsed test
+  ansible.builtin.assert:
     that:
       - "result.changed == false"
       - "result.parsed == parsed"
 
-- name: Idempotence - parsed
-  cisco.nxos.nxos_static_routes: *parsed
-  register: result
-
 - ansible.builtin.assert:
```

Comment:
This assert seems to be redundant since the previous task is (correctly) removed.